pkg/client/ipam.go

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright 2016-2017 Authors of Cilium
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+package client
+
+import (
+	"github.com/cilium/cilium/api/v1/client/ipam"
+	"github.com/cilium/cilium/api/v1/models"
+)
+
+const (
+	AddressFamilyIPv6 = "ipv6"
```

Comment:
1) This is a common const between the daemon and the client right?

2) Maybe create a new type?
```
type AddressFamily string
const(
    AddressFamilyIPv6 = AddressFamily("ipv6")
...
```
that way `IPAMAllocate(family string)` becomes `IPAMAllocate(family AddressFamily)`